### ISSUE TITLE
Hello. How is the deeplabV3plus implemented by mmsegmentation inconsistent with the paper?Besides, why is there an auxiliary_ head.loss_ decode？

### ISSUE THREAD

**Issue by daigang896** [open-mmlab/mmsegmentation#770]

**Comment by Junjun2016** [comment]
Hi @daigang896 
The OS (output stride) is different from the original paper.
OS is 8 in MMSegmentation while os is 16 in the original paper.

We also add auxiliary loss for consistent settings.

**Comment by daigang896** [comment]
I see. Another problem is that when the category data is extremely unbalanced, the default crossentropyloss does not work well. What should I do?

**Comment by Junjun2016** [comment]
Try weighted cross-entropy or dice loss.

**Comment by daigang896** [comment]
Thanks.
